docs/guides/primitives.mdx

```diff
@@ -17,45 +17,70 @@ In this context, a *primitive* is the smallest processing instruction, the simpl
 one can create something useful for a given abstraction level.
 
 The recent progress in quantum computing has increased the need to work at higher levels of abstraction.
-As we move toward larger QPUs (quantum processing units) and more complex workflows, the focus shifts from interacting with individual
-qubit signals to viewing quantum devices as systems that perform tasks we need.
+As the field moves toward larger quantum processing units (QPUs) and more complex workflows, the focus shifts from interacting with individual
+qubit signals to viewing quantum devices as systems that perform necessary tasks.
 
-The two most common tasks quantum computers are used for are sampling quantum states and calculating expectation values.
-These tasks motivated the design of the Qiskit primitives: Sampler and Estimator.
+The two most common tasks for quantum computers are sampling quantum states and calculating expectation values.
+These tasks motivated the design of the Qiskit primitives: **Estimator** and **Sampler**.
+
+- Estimator computes expectation values of observables with respect to states prepared by quantum circuits.
+- Sampler samples the output register from quantum circuit execution.
 
 In short, the computational model introduced by the Qiskit primitives moves quantum programming one step closer
 to where classical programming is today, where the focus is less on the hardware details and more on the results
 you are trying to achieve.
 
-## Implementation of Qiskit primitives
+## Primitive definition and implementations
+
+There are two types of Qiskit primitives: the base classes, and their implementations. The Qiskit primitives are defined by open-source primitive base classes that live in the Qiskit SDK (in the  [`qiskit.primitives`](/api/qiskit/primitives) module). Providers can use these base classes to derive their own Sampler and Estimator implementations.
```

Comment:
I think it would be worth adding one more sentence to this paragraph that says that most users will interact with provider implementations, and only provider developers need to worry about the base primitives